tsconfig.json

```diff
@@ -39,6 +39,7 @@
     // "baseUrl": "./",                       /* Base directory to resolve non-absolute module names. */
     // "paths": {},                           /* A series of entries which re-map imports to lookup locations relative to the 'baseUrl'. */
     // "rootDirs": [],                        /* List of root folders whose combined content represents the structure of the project at runtime. */
+    "resolveJsonModule": true,
```

Comment:
I wanted to apply this to only the `content/test` folder so we could `import` mock JSON responses directly. I put at tsconfig.json file in that folder that extends this one and added this one rule, but it didn't seem to work. I don't _think_ this affects the runtime code, but I'm open to other suggestions as to how to scope it to just the test files.